game.js

```diff
@@ -171,7 +186,61 @@ class Game extends Screen {
 	}
 
 	drawBackground() {
-		if (this.bgImage) background(this.bgImage, 100)
+		if (!this.bgImage) return;
+		var modifiedImage = this.bgImage;
```

Comment:
hmm... this feels like a macro problem.  The background should probably be drawn at a higher level (in the main loop) rather than down here in the Game/screen because this might have to be done on all screens.  We could then detect the [windowResized](https://p5js.org/reference/#/p5/windowResized) in one place and make the adjustment there.  The individual screen/game's job would be to hold the bg image/color (if any)

Reply:
Yes, that's a good idea